cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -337,18 +344,27 @@ private static ImagePullPolicy parseImagePullPolicy(String imagePullPolicyEnvVar
         return imagePullSecrets;
     }
 
-    private static Labels parseOperatorNamespaceLabels(String operatorNamespaceLabelsString) {
-        Labels operatorNamespaceLabels = null;
+    /**
+     * Parse labels from String into the Labels format. This is used to parse the custom resource labels and the
```

Comment:
I would avoid adding "This is used to ... " because at some point we could use this method for parsing some other labels and if the JavaDoc is not updated it would be stale.